### PR TITLE
Added a README for devices

### PR DESCRIPTION
**Pull request by elandini84**
Added a README markdown file to `src/devices` folder that shows the current  development status of all the devices in the folder.

### CODE REVIEW

**Comment by traversaro** [comment]
Is this linked in some way from the main README or in the doxygen documentation? In my experience README in nested directories are quite hard to find, unless they are linked directly or indirectly in the main documentation entry points.

**Comment by elandini84** [comment]
> Is this linked in some way from the main README or in the doxygen documentation? In my experience README in nested directories are quite hard to find, unless they are linked directly or indirectly in the main documentation entry points.

This is a really god point. This file is aimed directly to developers and not to a generic YARP user. Therefore I do not know if it should be in the main README or in the doxygen documentation. Every suggestion is welcome 😊

**Comment by traversaro** [comment]
Ah sorry, I misunderstood its role then.

**Comment by drdanz** [comment]
Link to the rendered README: https://github.com/elandini84/yarp/tree/dox_deviceReadme/src/devices

File: src/devices/README.md [hunk 1]
```diff
@@ -0,0 +1,330 @@
+## Yarp devices
```
Comment (drdanz) [inline src/devices/README.md:1]:
```suggestion
## YARP devices
```

**Review by drdanz — CHANGES_REQUESTED** [review]
Can you please fix this and rebase over latest master?

**Pull request merged**